app/assets/javascripts/application.js.erb

```diff
@@ -319,6 +263,12 @@ class _LobstersFunction {
     fetchWithCSRF("/stories/" + li.getAttribute("data-shortid") + "/" + act, {method: 'post'});
   }
 
+  modalRemoval() {
+    document.querySelector("#flag_dropdown").remove();
+    document.querySelector("#modal_backdrop").remove();
+    document.querySelector('.flag-wrapper').remove();
```

Comment:
If there can only be one of these (implied by the use of `querySelector` instead of `querySelectorAll`), use an id rather than a class.